src/components/board/board.test.jsx

```diff
@@ -1,17 +1,18 @@
 import React from 'react';
 import { shallow, mount } from 'enzyme';
+
 import Board from './board';
 
 describe('<Board/>', () => {
   it('checks if component exists', () => {
-    const squares = Array(9).fill(null);
+    const squares = Array(9).fill('');
```

Comment:
Create more real data with the squares array here maybe ever first one a 0 and second one an X.
And add a test to check the value of one of the squares that get mounted to make sure it has the correct value